packages/nouns-webapp/wagmi.config.ts

```diff
@@ -0,0 +1,132 @@
+import { defineConfig } from '@wagmi/cli';
```

Comment:
wondering if this shouldn't go on @nouns/sdk instead. it has value as a standalone package

Reply:
might work but it breaks the current sdk. we need a roadmap for @nouns/assets and @nouns/sdk and that’s outside this PR. i do agree some of this code should live in reusable packages.